### PR TITLE
refactor(ui5-radiobutton): API enhancements

### PR DESCRIPTION
**Pull request by kskondov**
Part of #3107

BREAKING_CHANGE: Changed "selected" property to "checked". Changed "select" event to "change"

### CODE REVIEW

File: packages/main/test/pages/RadioButton.html [hunk 1]
```diff
@@ -60,23 +60,23 @@
...
-		<ui5-radiobutton id="myRb9" readonly text="read only" selected></ui5-radiobutton>
-		<ui5-radiobutton id="myRb10" disabled text="disabled" selected></ui5-radiobutton>
+		<ui5-radiobutton id="myRb9" readonly text="read only" checked></ui5-radiobutton>
+		<ui5-radiobutton id="myRb10" disabled text="disabled" checked></ui5-radiobutton>
 		<ui5-radiobutton id="myRb11" value-state="Warning" text="warning"></ui5-radiobutton>
 		<ui5-radiobutton id="myRb12" value-state="Error" text="error"></ui5-radiobutton>
-		<ui5-radiobutton id="myRb13" value-state="Warning" selected text="warning"></ui5-radiobutton>
-		<ui5-radiobutton id="myRb14" value-state="Error" selected text="error"></ui5-radiobutton>
+		<ui5-radiobutton id="myRb13" value-state="Warning" checked text="warning"></ui5-radiobutton>
+		<ui5-radiobutton id="myRb14" value-state="Error" checked text="error"></ui5-radiobutton>
```
Comment (fifoosid) [inline packages/main/test/pages/RadioButton.html:79]: Update the event subscription below as well. That's why the build is failing

**Review by fifoosid — CHANGES_REQUESTED** [review]

File: packages/main/src/RadioButton.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ const metadata = {
 		 * @defaultvalue false
 		 * @public
 		 */
-		selected: {
+		checked: {
```
Comment (ilhan007) [inline packages/main/src/RadioButton.js:73]: Determines whether the component is selected or not.

->

**Defines** whether the component is **checked** or not.

**Review by ilhan007 — APPROVED** [review]

File: packages/main/src/RadioButton.js (hunk 2 above)
Comment (gmkv) [reply packages/main/src/RadioButton.js:73]: Done

**Review by ilhan007 — APPROVED** [review]

**Review by gmkv — APPROVED** [review]
LGTM. Another change will follow to rename the component to `ui5-radio-button` to align it with the rest of the *-button controls.

**Pull request merged**